unity/Assets/Scripts/PhysicsSceneManager.cs

```diff
@@ -173,6 +173,16 @@ public class PhysicsSceneManager : MonoBehaviour {
         }
     }
 
+    // public void MakeBreakableAndSleep() {
+    //     foreach (SimObjPhysics sop in GameObject.FindObjectsOfType<SimObjPhysics>()) {
+    //         if(sop.IsBreakable) {
+    //             sop.GetComponent<Rigidbody>().Sleep();
+    //             sop.GetComponentInChildren<Break>().Unbreakable = false;
+    //         }
+
+    //     }
+    // }
+
```

Comment:
Is this necessary?

Reply:
no, this was leftover from a previous attempt at a fix. I'll remove it.